# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/credentialmanagement/identity/WebIdentityChild.h
# Commit: 63d9972bab1f
# Full Hash: 63d9972bab1f267a940ec30c7cea7a4c41e54fa6
# Author: Alexandru Marc <amarc@mozila.com>
# Date: 2025-06-06 08:52:03
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing leaks @ mochitest.toml
#   
#   This reverts commit a97cb1d6c08bc38e8644bfc916298b847dbac3a1.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r 801e25cb4bfd -r 63d9972bab1f dom/credentialmanagement/identity/WebIdentityChild.h
--- a/dom/credentialmanagement/identity/WebIdentityChild.h	Thu Jun 05 18:51:36 2025 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,35 +0,0 @@
-/* -*- Mode: C++; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
-/* vim: set ts=8 sts=2 et sw=2 tw=80: */
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#ifndef mozilla_dom_WebIdentityChild_h
-#define mozilla_dom_WebIdentityChild_h
-
-#include "mozilla/dom/PWebIdentity.h"
-#include "mozilla/dom/PWebIdentityChild.h"
-
-namespace mozilla::dom {
-
-class WebIdentityHandler;
-
-class WebIdentityChild final : public PWebIdentityChild {
- public:
-  NS_INLINE_DECL_REFCOUNTING(WebIdentityChild, override);
-
-  WebIdentityChild() = default;
-
-  void ActorDestroy(ActorDestroyReason why) override;
-
-  void SetHandler(WebIdentityHandler* aHandler);
-
- private:
-  ~WebIdentityChild() = default;
-
-  WebIdentityHandler* mHandler{};
-};
-
-}  // namespace mozilla::dom
-
-#endif  // mozilla_dom_WebIdentityChild_h